pyproject.toml

```diff
@@ -18,7 +18,8 @@ dvc = "^3.50.1"
 dvc-gdrive = "^3.0.1"
 tensorflow-intel = { version = "^2.13.0", platform = "win32" }
 tensorflow-io-gcs-filesystem = [
-    { version = "< 0.32.0", markers = "platform_system == 'Windows'" }
+    {version = "< 0.32.0", platform ="win32"},
+    {platform = "darwin", version = "~0.36.0"}
```

Comment:
Poetry wasn't compiling on MAC so made this change. Can someone with windows check if things work fine on their system as well?

Reply:
will do